source/patterns/01-molecules/contextual-data/contextual-data.yaml

```diff
@@ -0,0 +1 @@
+doi: 10.5468/ogs.2016.59.1.1
```

Comment:
Suggest we use `10.5555/12345678` (https://www.crossref.org/blog/doi-like-strings-and-fake-dois/).